src/compiler.c

```diff
@@ -5137,6 +5137,9 @@ static void CompFunc(Obj func)
     /* emit the code to switch to a new frame for outer functions          */
     Emit( "\n/* allocate new stack frame */\n" );
     Emit( "SWITCH_TO_NEW_FRAME(self,%d,0,oldFrame);\n",NHVAR_INFO(info));
+    if(NHVAR_INFO(info) > 0) {
```

Comment:
```suggestion
    if (NHVAR_INFO(info) > 0) {
```

Reply:
Thought I'd formatted, now clang-formatted throughout (which makes a couple of other tiny changes)